dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/analytics/enrollment/EnrollmentQueryTest.java

```diff
@@ -287,12 +287,13 @@ public void queryWithProgramAndRepeatableProgramStage() {
 
     validateRowContext(response, 0, 3, "ND");
     validateRowContext(response, 1, 3, "ND");
-
     validateRow(
-        response, 0, List.of("Ioxjc2KBjWd", "Sierra Leone / Bo / Kakua / Fengehun MCHP", "", ""));
+        response,
```

Comment:
As we discussed, this fix is not needed. You can revert it completely, and get what I just merged into 2.40:
`https://github.com/dhis2/dhis2-core/pull/15462/files#diff-f5d374aef17fa25716cb6de4815c4d82864e55c258c9cbdefe6f63ff2ac7a1f2R242`

Reply:
Yep, it was too late :-)